store/tikv/coprocessor.go

```diff
@@ -639,7 +651,7 @@ func (worker *copIteratorWorker) logTimeCopTask(costTime time.Duration, task *co
 	var detail *kvrpcpb.ExecDetails
 	if resp.Cop != nil {
 		detail = resp.Cop.ExecDetails
-	} else if resp.CopStream != nil {
+	} else if resp.CopStream != nil && resp.CopStream.Response != nil {
```

Comment:
Maybe a better fix is make CopStream.Response always not nil, and report error when it's used ?